### PR TITLE
Add MEDAS

### PR DESCRIPTION
**Pull request by oxygene76**
Add MEDAS to Frontier

### CODE REVIEW

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
Good start, but insufficient--it also requires the chain be added to chain-infos.ts, and the token logo (svg preferred) should be added to public/tokens/ folder.

File: packages/web/config/ibc-assets.ts [hunk 1]
```diff
@@ -963,6 +963,12 @@ export const IBCAssetInfos: (IBCAsset & {
     sourceChannelId: "channel-326",
     destChannelId: "channel-5",
     coinMinimalDenom: "stustars",
+  },
+   {
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:967]: please move to the bottom of the list

Comment (oxygene76) [reply packages/web/config/ibc-assets.ts]: Thank you, just added the chain to chain-infos.ts and added the token logos.

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -2346,6 +2346,30 @@ const chainInfos = (
...
+        coinType: 118,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("medas"),
+      currencies: [
+        {
+          coinDenom: "MEDAS",
+          coinMinimalDenom: "umedas",
+          coinDecimals: 6,
+          // coinGeckoId: "medasdigital",
+          coinGeckoId: "pool:umedas",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:2364]: Shall we use pool 859 to define this 'id' in the prices.ts file? With this, the MEDAS token will have a price to display in the frontend.

Comment (JeremyParish69) [reply packages/web/config/chain-infos.ts:2364]: Yes please

Comment (oxygene76) [reply packages/web/config/chain-infos.ts:2364]: Ok, added to price.ts

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
Can we please add the gasPriceStep?
Also, I'm, getting CORS errors; please ensure the endpoints don't block the site.

**Comment by oxygene76** [comment]
of course, added gasPriceStep.. Had a fast check for cors, will resolve tomorrow. Thank you

**Pull request merged**